SketchyDevelopment/Game.cs

```diff
@@ -0,0 +1,111 @@
+                while (true) {                      // 2. will eternally ask the user util they
+                                                    // give a valid answer
+                    try {
+                        reply = _RespondYesOrNo();
+                        break;
+                    }
+                    catch (ArgumentException ex) {  // if the reply is invalid, then
+                                                    // "_RespondYesOrNo()" will pull an
+                                                    // ArgumentException before "break" is run
+                        Console.WriteLine(ex.Message);
+                    }
+                }
+                if (reply == "no") {                // 3. if user responded "no", then exit this
+                                                    // function
+                    return;
+                }
+                                                    // else do nothing (and this forever loop is
+                                                    // repeated)
+            }
+        }
+        /// <summary>
+        /// This private method asks the use to respond to a yes/no question and will return "yes",
+        /// "no" or an "ArgumentException" if the responce can't be rounded to either of these
+        /// answers.
+        /// </summary>
+        /// <returns>
+        /// Rounds up the user's answer to "yes" or "no" and returns "yes" or "no"
+        /// </returns>
+        /// <exception cref="ArgumentException">
+        /// Will raise an "ArgumentException" if the input isn't "n", "y", "no" or "yes"
+        /// </exception>
+        private string _RespondYesOrNo() {
+            string reply = Console.ReadLine();      // get response from user
+            if (reply == string.Empty) {            // throw an error if they give no reply
+                throw new ArgumentException("ERROR: Please give an input");
+            }
+            reply = reply.ToLower();                // convert string to lower case letters to
+                                                    // simplify the coming "if" statements
+            if (                                    // if responce can be rounded to "yes" then
+                reply == "y" ||                     // return "yes"
+                reply == "yes"
+                ) {
+                    return "yes";
+                }
+            else if (                               // if responce can be rounded to "no" then
+                reply == "n" ||                     // return "no"
+                reply == "no"
+                ) {
+                    return "no";
+                }
+            else {                                  // else it couldn't be rounded to anything, so
+                                                    // return/throw an error
+                throw new ArgumentException("ERROR: Please input \"yes\" or \"no\"");
+            }
+        }
+    }
+}
```

Comment:
Have the requirements been met?
Yes you have covered everything that has been asked from you
Is the code formatted using the Style Guidelines correctly?
Yes everything is in the correct Style Guide lines
Is the code easy to read?
Yes everything is neat, used suitable names and is suitably commented but the continues game is hard to interrupt but that just might just be me
Are different errors handled correctly?
Yes you have because everything is covered in a try and catches but for your _RespondYesOrNo() method I recommend using Trim(); ToLower(); to get rid of the white spaces and make it lower case for less type errors from the user and so they don't have to type it again